tests/jballerina-unit-test/src/test/resources/test-src/expressions/binaryoperations/integer_range_operators.bal

```diff
@@ -1,3 +1,5 @@
+public type Label int; // If the TypeHashVisitor takes in type.flags of `Lable` type, int range tests will fail.
+
```

Comment:
Shall we move this out to a separate file? And run and assert an iteration maybe?
```suggestion
public type Label int; // If the TypeHashVisitor takes in type.flags of `Label` type, int range tests will fail.

```